docs/source/use_cases/index.rst

```diff
@@ -1,25 +1,96 @@
+Iroha applications 
+===================
+
+Distributed ledger technology, and blockchain specifically,  receive massive attention in recent years. 
+This area is constantly evolving with many various applications popping up every day. 
+Although, we see a massive adoption is it still far from widespread use. 
+
+Current blockchain system lack of governance and usage simplicity.  
+While the technology itself showed to be a disruptive and promising, some basic functionality and lack of clear and simple interface complicates the adoption of technology. 
+Iroha team carefully studied all current approaches for creating the best solution according to the needs of clients. 
+
+
+The main focus of the Iroha is to give soft transition of current centralized applications to multiple application in a shared decentralized space while maintaining the best of two worlds.   
+Iroha does not completely removes intermediaries, but rather disintermediate current business process. 
+Such disintermediation will benefit both users and service providers enchanting business with new channels and information. (weak)   
+Iroha puts client and consumer as a priority. 
+Iroha differs from other similar systems in following: 
+- Possibility to create a light client tracing only related transactions It is possible to make a lightweight client storing only related transactions.   
+- Clear Transaction/Query separation with embedded permissions and role-based access control 
+- Native support for mobile client 
+- Fast and robust  asynchronous consensus 
+- Possibility to tune Iroha to your needs 
+- Narrowed attack space due to the restriction of smart contracts to secure core commands
+- Possibility to implement middleware software for your application   
+
+
+Iroha has an embedded role-based control system with rich permissions set.
+Each command or query in Iroha is verified according to the defined business rules.   
+Permissions in Iroha allow restricting data usage only to a limited set of users reducing the probability of privacy leakage. 
+Moreover, Iroha users can store either encrypted data or even only hash for the data integrity.  
+
+Each asset in Iroha has a specific verified creator. 
+The asset creation and distribution process are totally transparent for the verifiers. 
+We simplified asset creation to just only one command, allowing to tokenize physical items and transfer ownership to other Iroha users. 
+ 
+
+Iroha uses SWIG to connect interfaces written C++ with a variety of high-level programming languages. 
```

Comment:
written in C++

Reply:
connect C++ external interfaces with a variety of high-level programming languages.